src/file/upload.repository.ts

```diff
@@ -13,7 +13,9 @@ export class UploadRepository {
    * @param upload - the upload to be created.
    */
   static create(upload: Partial<IUpload>): Promise<IUpload> {
-    upload.ownerID = upload.ownerID ? upload.ownerID : 'null';
+    if (!upload.ownerID) {
```

Comment:
Should be `if (upload && !upload.ownerID) {...}`, since `upload` could be `undefined` or `null`.

Reply:
+1

Reply:
Then there should also be a check for the case when upload is `undefined` or `null`.
If that is the case there should be a Server Error.